test/fixtures/autofix/jQueryNativeReplacements.js

```diff
@@ -0,0 +1,62 @@
+sap.ui.define(["sap/ui/thirdparty/jquery"], async function (jQuery) {
+	// https://github.com/SAP/ui5-linter/issues/590
+
+	// TODO: Requires couple runs!
+	// 1. Replace global "new sap.base.i18n.ResourceBundle()"
+	// 2. Replace the code with the following:
+	// -> var isBundle = new ResourceBundle() instanceof ResourceBundle;
+	var isBundle = jQuery.sap.resources.isBundle(new sap.base.i18n.ResourceBundle());
+
+	var isStandAlone = jQuery.device.is.standalone;
+	var retinaDisplay = jQuery.support.retina;
+	var startsWithH = jQuery.sap.startsWith("Hello", "H");
+	var startsWithH2 = jQuery.sap.startsWith();
+	var startText = "Hello";
+	var startsWithLetter = "h";
+	var startsWithH3 = jQuery.sap.startsWith(startText, "H");
+	var startsWithH3 = jQuery.sap.startsWith(startText, startsWithLetter);
+	var startsWithHOrh = jQuery.sap.startsWithIgnoreCase("Hello", "h");
+	var startsWithHOrh2 = jQuery.sap.startsWithIgnoreCase(startText, startsWithLetter);
+	var endsWithY = jQuery.sap.endsWith("Hello Y", "Y");
```

Comment:
There are special cases to be handled (see failing QUnit tests in openui5).

Reply:
Thanks!